Makefile

```diff
@@ -1,18 +1,23 @@
 # Go parameters
-GOCMD=go
-GOBUILD=$(GOCMD) build
-GOCLEAN=$(GOCMD) clean
-GOTEST=$(GOCMD) test
-GOGET=$(GOCMD) get
-GOINSTALL=$(GOCMD) install
+GOPATH ?= $(HOME)/go
+GOCMD := go
+GOBUILD := $(GOCMD) build
+GOCLEAN := $(GOCMD) clean
+GOTEST := $(GOCMD) test
+GOVET := ${GOCMD} vet
+GOLINT := golint
```

Comment:
this breaks my env:
```
martin@mayak:~/go/src/github.com/innosat-mats/rac-extract-payload$ make lint
go vet ./...
golint ./...
make: golint: Command not found
Makefile:15: recipe for target 'lint' failed
make: *** [lint] Error 127
```

I have golint installed and previous solution did work. Why doesn't it work for you?